libs/dh/shared/ui-search/tsconfig.spec.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["jest", "node"]
```

Comment:
suggestion: let's add `"target": "es2016"` here similar to other `tsconfig.spec.json` files, e.g. "\libs\dh\shared\ui-paginator\tsconfig.spec.json". This is required after updating some dependencies a couple of weeks ago.

Reply:
Fixed